README.md

```diff
@@ -207,13 +207,13 @@ The `compile` command is optimized to build only modified files and is fast.
 
 To create a new pattern, please follow these steps:
 
-1. Under lib create a folder for your pattern, such as `<pattern-name>-construct`. If you plan to create a set of patterns that represent a particular subdomain, e.g. `security` or `hardening`, please create an issue to discuss it first. If approved, you will be able to create a folder with your subdomain name and group your pattern constructs under it. 
+1. Under lib create a folder for your pattern, such as `<pattern-name>`. If you plan to create a set of patterns that represent a particular subdomain, e.g. `security` or `hardening`, please create an issue to discuss it first. If approved, you will be able to create a folder with your subdomain name and group your pattern constructs under it. 
 2. Blueprints generally don't require a specific class, however we use a convention of wrapping each pattern in a plain class like `<Pattern-Name>Construct`. This class is generally placed in `index.ts` under your pattern folder. 
 3. Once the pattern implementation is ready, you need to include it in the list of the patterns by creating a file `bin/<pattern-name>.ts`. The implementation of this file is very light, and it is done to allow patterns to run independently.
 
 Example simple synchronous pattern:
 ```typescript
-import SingleNewEksOpenSourceobservabilityConstruct from '../lib/single-new-eks-opensource-observability-construct';
```

Comment:
Why is `SingleNewEksOpenSourceobservabilityConstruct` still having `Construct` ? We want to totally get rid of word Construct with `Pattern`. Also we dont want `-construct` to be removed. We would rather want it to be renamed to `-pattern`.